examples/jetstream/limits-stream/python/main.py

```diff
@@ -0,0 +1,74 @@
+import os
+import asyncio
+import time
+
+import nats
+from nats.errors import TimeoutError, NoRespondersError
+
+
+servers = os.environ.get("NATS_URL", "nats://localhost:4222").split(",")
+
+
+async def main():
+    
+    # Connect to NATS server
+    nc = await nats.connect(servers=servers)
+    
+    # Get a context to produce and consume messages from NATS JetStream
+    js = nc.jetstream()
+    
+    # Create a stream named 'events' and with subjects matching 'events.*'
+    # 'events' will be a default stream that all events will be sent to
+    # Storage parameter can be set to 'NONE' for no storage, 'FILE' for file based storage, or 'MEMORY' for memory based storage
+    await js.add_stream(name='events', subjects=['events.*'], storage='MEMORY')
+    
+    # Publish 6 messages to the JetStream
+    await js.publish('events.page_loaded',b'')
+    await js.publish('events.mouse_clicked',b'')
+    await js.publish('events.mouse_clicked',b'')
+    await js.publish('events.page_loaded',b'')
+    await js.publish('events.mouse_clicked',b'')
+    await js.publish('events.input_focused',b'')
+    print("published 6 messages","\n")
+
+    # Check the number of messages in the stream using streams_info
+    # StreamState includes the total number of messages in the stream
+    print(await js.streams_info(),"\n")
+
+    # Update the 'events' stream to have a maximum of 10 messages
+    await js.update_stream(name='events', subjects=['events.*'], max_msgs=10)
```

Comment:
Unlike the Go example, we are not publishing 12 messages, only six above so the max messages limit will not have any noticeable change to the stream info. I would suggest publishing six more to be consistent with the Go example and this first limit will then work.

Reply:
Thank you. I made a commit